packages/addons/service/tinc/source/resources/language/English/strings.po

```diff
@@ -1,70 +1,29 @@
 # Kodi Media Center language file
 # Addon Name: tinc
 # Addon id: service.system.tinc
-# Addon Provider: awiouy at gmail dot com
```

Comment:
`# Addon Provider: Team LibreELEC` to keep in sync with the "standard" (that get hopefully merged some day)

Reply:
Done